src/IRremoteESP8266.h

```diff
@@ -37,6 +37,7 @@
  * Vestel AC code by Erdem U. Altınyurt
  * Teco AC code by Fabien Valthier (hcoohb)
  * Mitsubishi 112 AC Code by kuchel77
+ * Kelon AC code by Davide Depau (Depau)
```

Comment:
[OPTIONAL] Please also feel free to add yourself to the `.github/Contributors.md` file too!